src/env.cc

```diff
@@ -298,7 +298,12 @@ bool Environment::EmitNapiWarning() {
 void Environment::EnvPromiseHook(v8::PromiseHookType type,
                                  v8::Local<v8::Promise> promise,
                                  v8::Local<v8::Value> parent) {
-  Environment* env = Environment::GetCurrent(promise->CreationContext());
+  auto context = promise->CreationContext();
+  // if the context is undefined (not a node context) then skip
+  if (context->GetEmbedderData(node::Environment::kContextEmbedderDataIndex)->IsUndefined()) {
```

Comment:
I think `make lint` should give you an error, because this line is longer than 80 characters?

Also, maybe look at the [C++ style guide](https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md), e.g. comments should ideally be capitalized and punctuated. :)

Reply:
both done :)